mapper/mapper.go

```diff
@@ -327,6 +327,13 @@ func CreateVersionsList(ctx context.Context, d dataset.Model, edition dataset.Ed
 
 		p.Data.Versions = append(p.Data.Versions, version)
 	}
+
+	for i, ver := range p.Data.Versions{
+		if ver.VersionNumber == latestVersionNumber {
+			p.Data.Versions[i].IsLatest = true
```

Comment:
Minor and picky point but once we have set the latest should we break out of the loop?